src/App.vue

```diff
@@ -1,11 +1,118 @@
 <template>
-  HELLO WORLD
+  <div id="main-container">
+    <h1>MODAL PROJECT</h1>
+    <input type="text" ref="input" />
+    <button @click="focusInput">Focus</button>
+    <br />
+    <button @click="modalHandler(true)">Open Modal</button>
+    <modal-component
+      v-if="showModal"
+      :title="`Sign up for the Giveaway!`"
+      :content="`Grab your ninja swag for half price!`"
+      :theme="theme"
+      @modalHandler="modalHandler"
+    >
+      <p :style="[theme === 'sale' ? { color: 'white' } : null]">
```

Comment:
use classes not inline-style @aayushbatra02

Reply:
ok sir